tests/e2e/configurer/chain/chain.go

```diff
@@ -31,6 +38,13 @@ type syncInfo struct {
 	SyncInfo status `json:"SyncInfo"`
 }
 
+const (
+	// waitUntilRepeatPauseTime is the time to wait between each check of the node status.
+	waitUntilRepeatPauseTime = 5 * time.Second
```

Comment:
what is the purpose of waiting between each query? I feel like it would be better to just spam the node until valid query instead of waiting potentially five extra seconds for the valid response. not a big deal though

Reply:
Spamming is prone to causing undetermined behavior. Although Tednermint doesn't have it, there might be rate-limiting on GitHub runners.

5 seconds might be an overkill though will reduce it in a follow-up PR.

Reply:
500ms applied

Reply:
Actually, seems that 500 ms is too fast. Times out before reaching success condition. I will try 2s and 40 times repeat